examples/example.go

```diff
@@ -14,4 +14,17 @@ func main() {
 
 	USD, GBP, EUR := coindesk.CurrentPrice()
 	fmt.Println(USD, GBP, EUR)
+	
+	var historyPrice []Price
```

Comment:
Don't declare and initialize in separate line.

`historyPrice := coindesk.HistoryPrice("2013-09-01", "2013-09-05")`

This will be good.

**OR**

Instead of `[]Price`, it should be `[]coindesk.Price`